RELEASING.md

```diff
@@ -120,3 +123,45 @@ github repository.
     $ git push upstream v$MAJOR.$MINOR.$PATCH
     $ git push upstream v$MAJOR.$MINOR.x
     ```
+
+## Deployment
+
+Deployment to Maven Central (or the snapshot repo) is for all of the artifacts
+from the project.
+
+### Branch
+
+Before building/deploying, be sure to switch to the appropriate branch or tag.
+For the current release use:
+
+```bash
+$ git checkout -b v$MAJOR.$MINOR.$PATCH
```

Comment:
Is the `-b` correct?  I thought that you would need to check out the commit at the tag `v$MAJOR.$MINOR.$PATCH` to build the release, but `-b` creates a branch named `v$MAJOR.$MINOR.$PATCH` at the current commit.